pages/common/crontab.md

```diff
@@ -28,9 +27,9 @@
 
 `0 10 * * * {{command_to_execute}}`
 
-- Sample job which runs every minute on the 3rd of April:
+- Sample job which runs every 10 minutes:
 
-`* * 3 Apr * {{command_to_execute}}`
+`*/10 * * * * {{command_to_execute}}`
```

Comment:
I don't really get what this example shows. This isn't a valid command right?

Reply:
There are already pre-existing versions of these commands, but no, you cannot run this command. It is just the format which should be used within the crontab command.

Reply:
How about a description like `Sample crontab entry, which runs a command every 10 minutes:`?